lib/Method/Also.pm6

```diff
@@ -7,8 +7,11 @@ module Method::Also:ver<0.0.3>:auth<cpan:ELIZABETH> {
     role AliasableClassHOW {
 
         method compose (Mu \o, :$compiler_services) is hidden-from-backtrace {
-            o.^add_method(.key, .value) for %aliases{o.^name}[];
-            nextsame;
+          for %aliases{o.^name}[] {
+            next unless $_;
+            o.^add_method(.key, .value);
```

Comment:
No reason. 

I had debugging information in that loop and it was just easier to delete it and leave the block rather than optimize it in case I had gotten something wrong.